### ISSUE TITLE
what version of terraform azure provider is used

### ISSUE THREAD

**Issue by boillodmanuel** [pulumi/pulumi-azure#377]
Hello, 
I didn't figure out what version is used? Is it detailed somewhere?
By the way, could we pin the version as it's recommended in https://github.com/terraform-providers/terraform-provider-azurerm?

Best regards,

**Comment by lukehoban** [comment]
You can see the versions of the upstream Terraform provider that map to each version of the Pulumi provider at https://github.com/pulumi/pulumi-azure/blob/master/CHANGELOG.md.

**Comment by boillodmanuel** [comment]
Thanks.
So I guess we can not fix a specific version (it's updated along with pulumi version

**Comment by boillodmanuel** [comment]
By the way, it could be worth to mention it in doc

**Comment by lukehoban** [comment]
Agreed - I’ll repurpose this issue to track including the version of the provider being documented in the documentation (both Pulumi version and upstream Terraform Provider version for providers based on Terraform Providers).
